1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,234 @@
+---
+RFC:  RFCnnnn
+Author:  travisez13
+Status:  Draft
+SupercededBy:  N/A
+Version: 0.1
+Area: Engine
+Comments Due: 6/30/2019
+---
+
+# `PowerShell Core` Policy
+
+## Motivation
+
+Consumers, developers, and enterprise system administrators should be able to flexibly and reliably configure PowerShell 7.
+
+## Acknowledgement
+
+I based this off of @iSazonov 's RFC, for just a slightly different purpose.
+[PR #111](https://github.com/PowerShell/PowerShell-RFC/pull/111)
+
+## Goals
+
+1. Specify how PowerShell 7 will deal with having both Windows PowerShell and PowerShell Core Group Policy.
+    - This is covered in [Policy settings Setting Fall-Back](#policy-settings-setting-fall-back).
+1. Correct how the `pwsh -settingsfile` switch behaves.
```

Comment:
```suggestion
1. Define how the `pwsh -settingsfile` switch behaves.
```

Reply:
Since it already behaves one way, that is not quite correct.  
```suggestion
1. Define how the `pwsh -settingsfile` switch should behave.
```